jetstream/cli.py

```diff
@@ -503,7 +519,11 @@ def convert(self, value, param, ctx):
 )
 
 bucket_option = click.option(
-    "--bucket", default="mozanalysis", help="GCS bucket to write to", required=True
+    "--bucket",
+    default="mozanalysis",
+    help="GCS bucket to write to",
+    required=False,
+    type=ClickNullableString(),
```

Comment:
What is the advantage of creating this type vs just letting it be the default STRING type with `required=False`?

Reply:
Mainly to maintain the current behaviour (which by default is to publish results to GCS). I feel like there are a couple of places where we invoke jetstream but don't explicitly set `--bucket` although we want results published to GCS.